### PR TITLE
Created homepage

### PR DESCRIPTION
**Pull request by WGillian**

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const NavBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
```
Comment (richardgill) [inline src/components/header.js]: % width and heights are generally not best practice in a flexbox world (unless forced).

Is there a way to make it work with pure flexbox?

**Comment by WGillian** [comment]
screen shot 2018-01-29 at 10 16 37

File: src/components/footer.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const FooterHeader = styled.h5`
+  flex-direction: column;
+  align-self: center;
+  padding-bottom: 20px;
+`
+
+export default () => {
+  return (
+    <FooterContainer>
+      <FooterItem>
```
Comment (altaudio) [inline src/components/footer.js]: This doesn't feel dry, can you create a `FooterItem` component that takes a header prop and an array of subheadings?

File: src/components/header.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  flex-direction: row;
+  justify-content: center;
+  margin-right: 60px;
+  padding-top: 30px;
+`
+
+export default () => {
+  return (
+    <NavBar>
+      <a href="/" title="home">
```
Comment (altaudio) [inline src/components/header.js]: Should be using React router for routing rather than `<a>` tags

**Pull request merged**